octokit.gemspec

```
@@ -6,7 +6,7 @@ require 'octokit/version'
 Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '>= 1', '< 3'
   spec.add_dependency 'sawyer', '>= 0.5.3', '~> 0.8.0'
```

sawyer 0.9.0 was released, which is blocking from updating to faraday 2.

The only changes betwen sawyer 0.8.0 and 0.9.0 are https://github.com/lostisland/sawyer/pull/67 and https://github.com/lostisland/sawyer/pull/72.

```suggestion
  spec.add_dependency 'sawyer', '>= 0.5.3', '~> 0.9.0'
```

---

@tisba Thanks for sharing the new release. 👍🏼